docs/source/fitting/fitfunctions/FmuF.rst

```diff
@@ -0,0 +1,41 @@
+.. _func-FmuF:
+
+====
+FmuF
+====
+
+.. index:: FmuF
+
+Description
+-----------
+
+A fitting function for muon-fluorine bonds in ionic crystals.
+
+.. math:: A(t)=A_0e^{-\frac{(\sigma t)^2}{2}}e^{-\lambda t}G(t)
+
+where,
+
+:math:`\lambda` (MHz) is the exponential decay rate,
+
+:math:`\sigma` (MHz) is the Gaussian decay rate,
+
+:math:`\omega_D` (MHz) is the dipolar interaction frequency,
```

Comment:
Would it make sense to call this the dipolar interaction angular frequency (or something similar). Currently `omega_D` and `freq_D` have the same description which could be slightly confusing

Reply:
True. Will add \omega = 2\pi\nu. In fact I need to change MHz to rads^-1 for omega. Thanks